DS implementations/QUEUES/Queue_Implementation.cpp

```diff
@@ -0,0 +1,108 @@
+// C++ program for array implementation of queue
+
+#include <bits/stdc++.h>
```

Comment:
Either do no include `bits/stdc++.h` or do not `use namespace std` otherwise there will ambiguity with container `queue` from STL